src/Components/CookieConsent.js

```diff
@@ -22,10 +22,13 @@ function CookieConsent() {
   return (
     <ReactCookieConsent
       buttonText="accept"
+      enableDeclineButton
+      declineButtonText="decline"
       disableStyles
       containerClasses="cookie-box d-lg-flex d-sm-flex align-items-center"
       contentClasses="cookie-content d-flex align-items-center"
       buttonClasses="cookie-button btn btn-primary ml-auto"
+      declineButtonClasses="cookie-button btn btn-secondary ml-auto mr-2"
```

Comment:
`btn-secondary` is the correct one here, but I noticed, that it does not look optimal.

Probably during the boostrap 3 to 4 conversion something went wrong with `btn-secondary`. The color of it should be `#fff`. So simply add `.btn-secondary` right along `.btn-primary` and `btn-success` in `index.scss` line number `392`.